### ISSUE TITLE
Image scaling after change 

### ISSUE THREAD

**Issue by bucknerns** [readthedocs/sphinx_rtd_theme#630]
### Problem
Image scaling changed to a stretched view after 0.2.5b2 instead of just scaled.

### Reproducible Project

Not a public doc

```rst

Class Design
------------

.. image:: design.jpg
   :scale: 100 %
   :align: left

```

#### Error Logs/Results

Place any images or error logs that might be able to help solve the issue here.

#### Expected Results

It should work the same scaling and keeping the ratio.

### Environment Info

- Python Version:2.7.12
- Sphinx Version:1.7.4 and 1.6.5
- RTD Theme Version: >= 0.3.0

**Comment by bucknerns** [comment]
Behavior changed after grunt run.
* 19b22eb - Run grunt (#522) (4 months ago) <Aaron Carlisle>

**Comment by bucknerns** [comment]
Turns out the problem is caused by the change of this line. Not sure if it is an intentional change or not. 
git diff .
diff --git a/sass/_theme_rst.sass b/sass/_theme_rst.sass
index 9293dc1..410af7f 100644
--- a/sass/_theme_rst.sass
+++ b/sass/_theme_rst.sass
@@ -17,7 +17,7 @@
   // Sphinx by default applies HxW style attributes to images. This fixes that oversite.
   img
     max-width: 100%
-    height: auto
+    height: auto !important
 
   div.figure
     margin-bottom: $base-line-height

**Comment by bucknerns** [comment]
Seems it was intentional.  
https://github.com/rtfd/sphinx_rtd_theme/pull/505
https://github.com/rtfd/sphinx_rtd_theme/issues/316

I found a workaround.
So the reason I did scale 100% is because for some reason adding scaling makes the image clickable
so the person viewing the docs can view the full size image.(Maybe it not being clickable by default is the real bug)
According to the rst docs scale should be uniform scaling of the image which it is currently not.  
When setting the width it will scale the image horizontally, which it does.

Working fixes

```
Class Design
------------

.. image:: design.jpg
   :width: 100 %

.. image:: design.jpg
   :target: _images/design.jpg

```
